### ISSUE TITLE
Server option in settings

### ISSUE THREAD

**Issue by Awemeter** [flybywiresim/installer#69]
**Is your feature request related to a problem? Please describe.**

Not related to a bug

**Describe the feature you'd like to see implemented**
In settings, there should be an option to switch between servers in case one gets really slow or goes down one day.

Discord username: Awemeter#7919

**Comment by nistei** [comment]
I don't think we should add this. We use a CDN with 27 PoPs. Very unlikely every PoP will go down.
